### PR TITLE
Cleanup space for MemSQL tests to work

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

**Comment by findepi** [comment]
cc @wendigo

**Review by wendigo — APPROVED** [review]

File: .github/bin/cleanup-node.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Space before cleanup"
+df -h
+
+echo "Removing redundant directories"
+sudo rm -Rf /opt/hostedtoolcache/go/
+sudo rm -rf /usr/local/lib/android
```
Comment (findepi) [inline .github/bin/cleanup-node.sh:10]: @wendigo 

any reason to have trailing / here and use -R,
and use -r in others ?

Comment (wendigo) [reply .github/bin/cleanup-node.sh]: none

**Review by losipiuk — APPROVED** [review]

**Comment by losipiuk** [comment]
CI (at least some): timeouts starting up PT env containers.

**Pull request merged**